man/systemd.network.xml

```diff
@@ -2529,6 +2529,11 @@ IPv6Token=prefixstable:2002:da8:1::</programlisting></para>
             Defaults to unset.</para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><varname>Interface=</varname></term>
```

Comment:
Maybe, `OutgoingInterface=`? In `[RoutingPolicyRule]` we already have `OutgoingInterface=`. So, I prefer `OutgoingInterface=` for consistency.